### DISCUSSION TITLE
Serenity. Extra it doesn't work in .net 6

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#6816] [Help]
I am working on serenity .net6 project
I want to add
Serenity. Extra
But it doesn't work

How to use (for .net 6+ and ES module)

Copy the Modules/_Ext folder from the SerExtraESM project and place it accordingly.
Copy the wwwroot/lib/_Ext folder from the SerExtraESM project
Add following line in Modules/Common/ScriptInit.ts Q.Config.rootNamespaces.push('_Ext');
If want to apply Serenity.Extra css then you could add line in appsettings.bundles.json "~/Modules/_Ext/ExtStyles.css"
See LanguageGrid.ts and LanguageDialog.ts for example.

**Comment by asilmertkan** [comment]
You can write Serenity-Extra repository for correct your issue. This is not the right place for Extra.

**Reply by edwardch** [reply]
https://github.com/dfaruque/Serenity.Extra/

**Comment by VictorTomaili** [comment]
This is not our official supported library
